src/collections/Events.ts

```diff
@@ -132,6 +132,31 @@ const Events: CollectionConfig = {
 				leaves: ['bold', 'italic', 'underline', 'strikethrough'],
 			},
 		},
+		{
+			name: 'devprops',
+			label: 'Developer properties',
+			labels: {
+				singular: 'Property',
+				plural: 'Developer properties',
+			},
+			type: 'array',
+			fields: [
+				{
+					name: 'key',
+					type: 'text',
+					required: true,
+				},
+				{
+					name: 'value',
+					type: 'json',
```

Comment:
```suggestion
					type: 'text',
```
This should be text, this isn't always JSON. Maybe it is in for this project, but for forwards-compatibility, this should be a plain text field and be parsed on the website side if it is known to be JSON.